python/pylibcudf/pylibcudf/tests/test_utilities.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+import pylibcudf as plc
+
+
+def test_is_ptds_enabled():
+    assert isinstance(plc.utilities.is_ptds_enabled(), bool)
```

Comment:
I wan't sure if there was a better way to test this. IIRC, we don't currently have a `--ptds` build in CI. Maybe once we can do, we can pass the expected value through as an environment variable or something.

But FWIW, this is showing `True` for a `--ptds` build locally.

Reply:
This is fine!